package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@anrok/mammoth",
-  "version": "0.3.0",
+  "version": "0.3.2",
```

Comment:
Why skip `0.3.1`?

Reply:
tl;dr: I'll set it to "0.3.3".

What happened:
1. The "package.json" has 0.3.2. (Bradley's change.)
2. When Bradley updated to 0.3.2, we must not have run `npm install` and so 0.3.2 never got copied to "package-lock.json".

But this reminds me that I never updated the version for my change. I'll set it to "0.3.3".
- I think the standard practice is to update the version when you do a release. But it looks like we have been doing it on every commit?
- I guess that's fine for now, but maybe something we want to look into making more precise.

Reply:
Edit: I'll actually do 0.4.0, since it's a backwards-incompatible change that could break users. (In our own codebase, we have to delete our `asQuoted` workaround, otherwise we could end up with things being quoted twice.)